packages/@uppy/companion/src/server/Uploader.js

```diff
@@ -655,7 +655,7 @@ class Uploader {
     const { client, options } = s3Options
 
     const params = {
-      Bucket: getBucket(options.bucket),
+      Bucket: getBucket(options.bucket, null, this.options.metadata),
```

Comment:
what is `options.metadata`? Is that per request?

Reply:
it's to align it with getKey: https://github.com/transloadit/uppy/pull/4770/files#diff-a9010bc2c261a8c62e0afb506f446adf868d77125c893f20e96a834afbfd0690R659